include/ck/tensor_operation/gpu/grid/gridwise_normalization_welford_variance.hpp

```diff
@@ -319,7 +319,7 @@ struct GridwiseNormalizationWelfordVariance_mk_to_mk
             });
 
             static_for<0, MThreadSliceSize, 1>{}([&](auto iM) {
-                auto divisor = 1 / __builtin_amdgcn_sqrtf(var_thread_buf(iM) + epsilon);
+                auto divisor = 1 / ck::math::sqrt(var_thread_buf(iM) + epsilon);
```

Comment:
Does these two ```sqrt``` produce the same ISA?

Reply:
Both ck::math::sqrt(f32) and __builtin_amdgcn_sqrtf() will use v_sqrt_f32_e32 
However, __builtin_amdgcn_sqrtf  only support f32.
In case of f64, we need to call __builtin_amdgcn_sqrt().
However, ck::math::sqrt(double) will not use v_sqrt_f64_e64. 
So, I will change the implemenetation of  ck::math::sqrt to call builtin function in next PR